### PR TITLE
Katana60

### PR DESCRIPTION
**Pull request by rominronin**
3 default keymaps, 1 default qwerty, 1 colemak variation, 1 'RominRonin' variation (colemak mod-dh).

### CODE REVIEW

**Comment by jackhumbert** [comment]
Cool! Would you mind updating the readme.md to match [this template](https://docs.qmk.fm/documentation_templates.html#keyboard-readmemd-template)? All makefiles are also ignored, so you can delete those :)

It looks like you have a few `REPLACE_WITH_YOUR_NAME ` in the headers still ;)

**Comment by rominronin** [comment]
Hey, thanks for taking the time to review the request Jack! I hope the updates are better now.

**Comment by rominronin** [comment]
ok, I added the latest readme updates.

File: keyboards/katana60/readme.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+A 60% keyboard with a symmetrical staggered layout, sold by CandyKeys.com.
+
+Keyboard Maintainer: [Baris Tosun](https://github.com/rominronin)  
+Hardware Supported: Katana60 PCB v1, Teensy 2.0  
+Hardware Availability: [CandyKeys.com](https://CandyKeys.com)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make katana60/default
```
Comment (jackhumbert) [inline keyboards/katana60/readme.md]: This should actually be `make katana60:default` :)

**Comment by jackhumbert** [comment]
Would you mind removing all of the `Makefile`s here? They're unneeded and ignored now in favor of the `rules.mk` :)

**Comment by rominronin** [comment]
Done!

**Comment by jackhumbert** [comment]
@rominronin Did you push up your commits? I'm not seeing any changes here yet.

**Comment by rominronin** [comment]
Should be merged in now, I'm getting the hang of this github pull-request aspect of git!

**Comment by jackhumbert** [comment]
Awesome! Thanks :)

**Pull request merged**